content/en/blog/2023-04-01-march-update/index.md

```diff
@@ -0,0 +1,440 @@
+additional generators like `cluster`, `gitRepository`, and `apiClient`
+offer enhanced functionality and customization. Weave GitOps Enterprise
+now supports raw templating for greater flexibility, and the Sandbox
+environments provide real-time visibility and testing capabilities
+for Kubernetes infrastructure.
+
+New additions to the PolicyConfig UI simplify policy management, and the
+SOPS Secrets features enable seamless GPG and AGE key management, making
+it easier than ever to create encrypted secrets.  Experience the benefits
+of adopting Weave GitOps by leveraging these powerful new features for
+Kubernetes environments.
+
+![Weave GitOps Enterprise v0.19](wge-0.19.png)
+
+#### Terraform-controller
+
+The team has been working on the new release of [Terraform
+controller](https://github.com/weaveworks/tf-controller). Bug fixes
+related to the GitOps dependency management have been landed in the
+main branch. So please feel free to try it out.
+
+The team has also been working closely with the Flux team to ensure
+that Terraform controller will support everything in the coming Flux GA.
+
+#### Flux Subsystem for Argo
+
+[Flamingo, the Flux Subsystem for
+Argo](https://github.com/flux-subsystem-argo/flamingo), for ArgoCD 2.6
+and Flux v0.41, has been released. Flamingo is the only tool that combines
+the best two GitOps technologies together. Flux and ArgoCD are both
+CNCF graduate projects.
+
+This new Flamingo version includes support for the following:
+
+- Flux v0.41 which comes with many features and enhancements,
+- Pack of new user interface features from ArgoCD 2.6
+
+Chanwit Kaekwasi, the main developer of Flamingo, is looking for feedback.
+If you use Flamingo and want to chat about how you use it, where you
+would like it to go or just want to give some feedback, please find him on
+the #flux channel on CNCF Slack. Thanks a lot in advance!
+
+#### VS Code GitOps Extension
+
+Version 0.24.0 of the [VS Code
+extension](https://github.com/weaveworks/vscode-gitops-tools) was released.
+This version introduces a new feature for the users of Weave GitOps Enterprise:
+Templates. Using Templates users can create complex GitOps configurations,
+workflows and pipelines with a simple UI. Templates must be enabled in the
+VS Code settings to be available. The [README
+file](https://github.com/weaveworks/vscode-gitops-tools#readme) includes
+further information.
+
+The team is continuing to work on UI and performance improvements. In the 0.24
+release, the Sources and Workloads views are now grouped by Namespaces and
+their details are presented in a consistent way.
+
+#### New additions to the Flux Ecosystem
+
+We are very happy to see Timoni joining the [Flux Ecosystem](/ecosystem). Timoni
+is a package manager for Kubernetes, powered by CUE and inspired by Helm. Timoni
+can be [used together with Flux](https://timoni.sh/gitops-flux/) to create a
+GitOps delivery pipeline for Timoni’s module instances.
```

Comment:
```suggestion
We are very happy to see Timoni joining the [Flux Ecosystem](/ecosystem).

[Timoni](https://github.com/stefanprodan/timoni) is a package manager for Kubernetes, powered by [CUE](https://cuelang.org) and inspired by Helm.

The Timoni project strives to improve the UX of authoring Kubernetes configs. Instead of mingling Go templates with YAML like Helm, or layering YAML on top of each-other like Kustomize, Timoni relies on cuelang's type safety, code generation and data validation features to offer a better experience of creating, packaging and delivering apps to Kubernetes.

Timoni can be [used together with Flux](https://timoni.sh/gitops-flux/) to create a
GitOps delivery pipeline for Timoni’s module instances.
```